### ISSUE TITLE
bug(material/checkbox): VoiceOver interprets the SVG inside a checkbox as an "empty unlabelled image"

### ISSUE THREAD

**Issue by zelliott** [angular/components#23338]
#### Reproduction

Steps to reproduce:
1. Go to https://material.angular.io/components/checkbox/overview on Chrome
1. Turn on VO.
1. Linearly navigate to and past the checkbox via VO + arrow keys.
1. At some point, you should hear "To get missing image descriptions etc etc... empty unlabelled image". The virtual cursor is on the SVG element.
 
#### Expected Behavior

The SVG element should be hidden from screen readers. The fix is likely as easy as adding `aria-hidden="true"` to the root SVG element.

#### Actual Behavior

The SVG element was accessible to screen readers and unlabelled.

#### Environment

- Angular: Latest
- CDK/Material: Latest
- Browser(s): Chrome (maybe all)
- Operating System (e.g. Windows, macOS, Ubuntu): Presumably all
